pilot/pkg/serviceregistry/kube/controller/controller.go

```diff
@@ -711,8 +710,6 @@ func (c *Controller) Services() ([]*model.Service, error) {
 		out = append(out, svc)
 	}
 	c.RUnlock()
-	sort.Slice(out, func(i, j int) bool { return out[i].Hostname < out[j].Hostname })
```

Comment:
removing this sort because it is used in debug and push context - push context resorts it by creation time. So not needed for that case. If debug wants I can put it back